frontend/src/metabase-types/api/search.ts

```diff
@@ -88,6 +88,7 @@ export interface SearchResult<
   pk_ref: FieldReference | null;
   table_schema: string | null;
   collection_authority_level: "official" | null;
+  type?: "instance-analytics" | "trash" | "remote-synced" | null;
```

Comment:
should this pull in `CollectionType`? Also, is this only for collections? if so, maybe the prop should be named to reflect that